### PR TITLE
Make SolidData conform to the expected DIDDocument

### PR DESCRIPTION
**Pull request by joncinque**
* Map between SolidData and DIDDocument
* Update SolidData to look like DIDDocument
* Update did-resolver to 3.0.1
NOTE: I wasn't able to update the driver because it depends on the npm package of solid-did-client, and not the local one.

### CODE REVIEW

File: client/src/service/resolve.ts [hunk 1]
```diff
@@ -16,12 +16,15 @@ export const resolve = async (identifier: string): Promise<DIDDocument> => {
...
   );
-  // TODO map solidData to DIDDocument
-  return {
-    '@context': 'https://w3id.org/did/v1',
-    id: identifier,
-    // @ts-ignore
-    temp: solidData,
-    publicKey: [],
-  };
+  if (solidData !== null) {
```
Comment (dankelleher) [inline client/src/service/resolve.ts:19]: probably we should just error out here, or return an empty response, right? Np real value in keeping the dummy value

**Review by dankelleher — APPROVED** [review]

File: client/src/service/resolve.ts (hunk 1 above)
Comment (joncinque) [reply client/src/service/resolve.ts:19]: Throwing is probably best -- I didn't want to change the signature

Comment (dankelleher) [reply client/src/service/resolve.ts:19]: ok I'll deal with it

**Review by dankelleher — APPROVED** [review]

File: client/src/service/resolve.ts (hunk 1 above)
Comment (dankelleher) [reply client/src/service/resolve.ts:19]: aah I see you did 👍

**Pull request merged**